src/controllers/home.js

```diff
@@ -0,0 +1,17 @@
+const getData = require('./../database/queries/getData');
+
+exports.get = (req, res) => {
+  getData.getProposals(1).then((result) => {
+    if (req.cookies) {
```

Comment:
@AhmedAl-Almi Is it possible to do a more specific check here, for example `if (req.cookies.token) {` or something like that?  If so, that might be safer (to prevent an unrelated request cookie causing the user to go to the login page).  Maybe it's not possible 🤷‍♂️

Reply:
Sure you right, but I do not finish from this file I used it to render home just as a test I will return for this code again and I added "jwt" in the browser manually to render home because of the login post #14  not finish. @Anies12  made it #90 but waiting for approval if you can make a review for this branch if you like, please.  :)